core/src/main/scala/kafka/utils/Time.scala

```diff
@@ -44,6 +46,8 @@ trait Time {
 
   def nanoseconds: Long
 
+  def hiResClockMs: Long = TimeUnit.NANOSECONDS.toMillis(nanoseconds)
```

Comment:
@ijuma Thank you for looking into this. Since MockTime.scala maintains only one time, shall I leave the common implementation above in Time.scala?